src/skeleton/mod.rs

```diff
@@ -0,0 +1,281 @@
+/// Skeleton structs
+/// Owns json::Animation
+
+mod error;
+mod timelines;
+mod animation;
+
+use json;
+use from_json;
+use std::collections::HashMap;
+use std::io::Read;
+use std::f32::consts::PI;
+use serialize::hex::FromHex;
+
+// Reexport skeleton modules
+pub use self::error::*;
+use self::timelines::*;
+use self::animation::*;
```

Comment:
Yes I agree. At the beginning I didn't really know what exactly was supposed to be publicly exposed. I start having a better idea now most things are in place so I'll change this too